docker/python-tester/testlib.py

```diff
@@ -73,6 +99,14 @@ def read_file(prev):
     
     return read_file
 
+def json_parse(prev):
+    try:
+        return Ok(json.loads(prev))
+    except ValueError as e:
+        return Err(f"couldn't parse input string as JSON. Trace: {e}")
+    else:
```

Comment:
The `else` branch in a `try-else` will be executed if no exception was raised. To handle "any other exception", you can use an empty `except`:

```suggestion
    except:
```